auth-server/src/services/auth.ts

```diff
@@ -86,6 +86,24 @@ export default class AuthService {
 	}
 
 	public async SignUp(email: string, password: string, username: string, name: string): Promise<AuthObjectType> {
+		let existing = await User
+			.query()
+			.where('username', username)
+			.first()
+
+		if (existing) {
+			throw new Error(`User with username: ${username} already exist. Please choose a different username or login.`)
```

Comment:
Should we use graphql `ForbiddenError` for both errors?

Reply:
using ForbiddenError